docs/content/cluster-scope/add_operator.md

```diff
@@ -0,0 +1,67 @@
+#  Adding an Operator to a Cluster
+
+There are two methods one can follow to add an Operator to a cluster:
+1. Install via Operator Lifecycle Manager (OLM)
+1. Manual install
+
+Installing via OLM is easier and often the preferred method of installation, but there are a few things to consider when choosing this install method. In order to install your operator via OLM you need to find it in either the [k8s community operators list][1] or [ocp community operators list][2]. If your operator is available in a different catalog source you will need to check if said catalog source is available on the target cluster.
+
+## Install an operator via OLM
+
+### Pre-requisites
+* Check if an operator bundle already exists for the desired operator [here][3] (if it does exist you can probably just include this bundle in the desired cluster overlay)
+* Basic knowledge of git and kustomize
+* Basic understanding of k8s and OCP/OLM
```

Comment:
can you move these 2 items at the top of the pre-req list -- if someone doesn't know kustomize, they can't tell if a bundle exists in an overlay :)